contracts/token/ERC20/StandardToken.sol

```diff
@@ -156,4 +156,41 @@ contract StandardToken is ERC20 {
     return true;
   }
 
+  /**
+   * @dev Internal function that mints an amount of the token and assigns it to
+   * an account. This encapsulates the modification of balances such that the
+   * proper events are emitted.
+   * @param _account The account that will receive the created tokens.
+   * @param _amount The amount that will be created.
+   */
+  function _mint(address _account, uint256 _amount) internal {
+    totalSupply_ = totalSupply_.add(_amount);
+    balances[_account] = balances[_account].add(_amount);
+    emit Transfer(address(0), _account, _amount);
+  }
+
+  /**
+   * @dev Internal function that burns an amount of the token of a given
+   * account.
+   * @param _account The account whose tokens will be burnt.
+   * @param _amount The amount that will be burnt.
+   */
+  function _burn(address _account, uint256 _amount) internal {
+    totalSupply_ = totalSupply_.sub(_amount);
```

Comment:
Do we want to do input validation on `amount > 0`, `account != 0`, and `balances[account] > amount`?

Reply:
I'm not sure about `balances[account] > amount`. I thought we were using `SafeMath` for this kind of thing, now that it uses `require`.

>Is this something we want to do?

I'm not sure.